.github/workflows/release.yml

```
@@ -27,6 +27,9 @@ jobs:
     needs: [dist]
     runs-on: ubuntu-latest
     if: github.event_name == 'release' && github.event.action == 'published'
+    environment: pypi
```

A nicer syntax could be
```suggestion
    environment:
        name: pypi
        url: https://pypi.org/p/cibuildwheel
```
which makes the URL show up in the workflow graph UI (and some other places). I like rendering the versioned URL even, when I have a version variable available...